index.html

```diff
@@ -382,6 +403,26 @@ <h1 class="headline-philosophy" align="center">Philosophy</h1>
         }
       })
     </script>
+    <script type="text/javascript">
+      $(document).ready(function(){
+          $.get(
+              "https://raw.githubusercontent.com/SUSE/machinery/master/NEWS",
+              {},
+              function(data) {
+                var latestEntry = data.split("##")[1].split("* ");
+                var heading = latestEntry[0]
+                $("#latest-release").text(heading.match(/\d+\.\d+.\d+/))
+                for (var i = 1; i < latestEntry.length; i++) {
+                  var issue = latestEntry[i].match(/gh#SUSE\/machinery#(\d+)/)
+                  if(issue) {
```

Comment:
I'd add a space after `if` because there is also one after `for`. That makes it more consistent.